### PR TITLE
[crio] Collect goroutine information

### PR DESCRIPTION
**Pull request by haircommander**
According to instructions in https://github.com/cri-o/cri-o/blob/main/tutorials/debugging.md

Signed-off-by: Peter Hunt <pehunt@redhat.com>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [x] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

File: sos/report/plugins/crio.py [hunk 1]
```diff
@@ -50,6 +50,12 @@ def setup(self):
             "crio config"
         ])
 
+        result = self.exec_cmd("kill -USR1 $(pidof crio)")
+        if result['status'] == 0:
```
Comment (pmoravec) [inline sos/report/plugins/crio.py:54]: Are you sure the `$(..)` will be properly evaluated? As when I test it on a mocked `crio` command, I always get `result` as:

    {'status': 1, 'output': 'kill: cannot find process "$(pidof"\nkill: cannot find process "crio)"\n', 'truncated': False}

Comment (TurboTurtle) [reply sos/report/plugins/crio.py:54]: Nack to bash-foo in plugins. Get the pid first, and if one exists, make another call to `kill`.

File: sos/report/plugins/crio.py [hunk 2]
```diff
@@ -50,6 +50,12 @@ def setup(self):
...
+            self.add_copy_spec([
+                "/tmp/crio-goroutine-stacks*.log"
+            ])
```
Comment (TurboTurtle) [inline sos/report/plugins/crio.py:57]: `add_copy_spec()` supports single strings, so no need for a list here. Glob expansion is done later on.

File: sos/report/plugins/crio.py (hunk 1 above)
Comment (haircommander) [reply sos/report/plugins/crio.py]: makes sense, updated

**Comment by haircommander** [comment]
updated for comments. The test failures don't look related, are they?

**Review by TurboTurtle — APPROVED** [review]
LGTM.

Test failure is unrelated and a known issue - restarted the job.

**Review by pmoravec — APPROVED** [review]

**Pull request merged**